admin-views/tribe-options-display.php

```diff
@@ -10,6 +10,18 @@
 	$template_options[$templates[$template]] = $template;
 }
 
+/**
+ * Filter the array of views that are registered for the tribe bar
+ * @param array array() {
+ *     Associative array of views
+ *
```

Comment:
"Array of views, where each view is itself represented by an associative array consisting of these keys:"

...too wordy?